src/components/Result/ResultContent/SongResult.tsx

```diff
@@ -33,6 +33,7 @@ const SongResult = ({
           <div className="hidden md:block">
             <ShareLinkButton
               trackShare={trackShare}
+              songDetails={songDetails}
```

Comment:
you missed adding `songDetails` to the `<ShareLinkButton/>` that appears on the mobile layout (Line 46 of the same file)

Reply:
ah nice catch